app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -154,47 +153,48 @@ class RootActivity : AppCompatActivity(), RootScreen {
                                         maxDate,
                                         initialDate,
                                         onDatePicked ->
-            val picker = DatePickerDialog(this)
+            val datePicker =
+                MaterialDatePicker.Builder.datePicker()
+                .setSelection(MaterialDatePicker.todayInUtcMilliseconds())
+                .build()
+            datePicker.show(supportFragmentManager, "datePicker")
+            datePicker.addOnPositiveButtonClickListener {
+                onDatePicked(LocalDate.ofEpochDay(it / 86400000))
```

Comment:
What's that magic number? You need to extract it as a constant and add KDoc to it. This will fix the Detekt error

Reply:
That's number of millisecond is a day. Added what to a const as suggested